### PR TITLE
fix(config): add Neo Coolcam Motion Sensor 2

### PR DESCRIPTION
**Pull request by bwosborne2**
fix(config): Add Neo Coolcam Motion Sensor 2
Also removed device from motion_sensor.

### CODE REVIEW

**Comment by AlCalzone** [comment]
This is supposed to fix #2738, correct?

Side note: The original file does not comply with the style guide, we should clean that up after this PR is merged.

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+				{
+					"label": "Enable Lux Level Function",
+					"value": 1
+				}
+			]
+		},
+		"9": {
+			"label": "Lux Level Changed Report",
+			"description": "How much the Lux Level must change",
+			"unit": "LUX",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"unit": "lux",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Sensitivity Level Setting",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "Sensitivity Level",
```
Also, the description is repetitive and should explain the values (e.g. 8 = low, 255 = high)

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+			"valueSize": 1,
+			"minValue": 8,
+			"maxValue": 255,
+			"defaultValue": 12,
+			"unsigned": true
+		},
+		"2": {
+			"label": "ON/OFF Duration Setting",
+			"description": "Determines how long the associated devices should stay ON status",
+			"unit": "Seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"unit": "seconds",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+			"description": "Defines the sensitivity of PIR sensor.",
+			"valueSize": 1,
+			"minValue": 8,
+			"maxValue": 255,
+			"defaultValue": 12,
+			"unsigned": true
+		},
+		"2": {
+			"label": "ON/OFF Duration Setting",
+			"description": "Determines how long the associated devices should stay ON status",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "On/Off Duration",
			"description": "Determines how long the associated devices should stay on",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+			"label": "Basic Set Level",
+			"description": "Basic Set Command will be sent when motion is triggered",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"unsigned": true
+		},
+		"4": {
+			"label": "Enable/Disable PIR Function",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "PIR Detector",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+			"options": [
+				{
+					"label": "Disable PIR Detector Function",
+					"value": 0
+				},
+				{
+					"label": "Enable PIR Detector Function",
+					"value": 255
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 255
				}
			]
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+		},
+		"5": {
+			"label": "Lux Level Set",
+			"description": "Set a lux level value which determines when the light sensor is activated.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "Light Sensor Brightness Threshold",
			"description": "The light sensor is active when the brightness is above this level",
```
Not sure about the description - is that correct or the other way round?

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+			"unit": "LUX",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"unit": "lux",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 100
+		},
+		"6": {
+			"label": "Re-trigger Interval",
+			"description": "Adjust the interval of PIR being re-triggered",
+			"unit": "Seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"unit": "seconds",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 8
+		},
+		"7": {
+			"label": "Light Sensor Polling Interval",
+			"description": "The Interval after which the Light Sensor Measure is polled.",
+			"unit": "Seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"unit": "seconds",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+			"description": "Set a lux level value which determines when the light sensor is activated.",
+			"unit": "LUX",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 100
+		},
+		"6": {
+			"label": "Re-trigger Interval",
+			"description": "Adjust the interval of PIR being re-triggered",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "PIR Re-trigger Interval",
			"description": "Must be less than the On/Off Duration",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+			"unit": "Seconds",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 8
+		},
+		"7": {
+			"label": "Light Sensor Polling Interval",
+			"description": "The Interval after which the Light Sensor Measure is polled.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json:84]: tbh I read the manual and still don't understand this. I think there's a better description

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+			"options": [
+				{
+					"label": "Disable Lux Level Function",
+					"value": 0
+				},
+				{
+					"label": "Enable Lux Level Function",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 60,
+			"maxValue": 36000,
+			"defaultValue": 180,
+			"unsigned": true
+		},
+		"8": {
+			"label": "Lux Level Function Enable",
+			"description": "0 = Off, 1 =On",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "Lux Level Function",
			"description": "When enabled, send Basic Set if brighness falls below threshold.",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 15]
```diff
@@ -0,0 +1,158 @@
...
+			"unit": "LUX",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 100,
+			"unsigned": true
+		},
+		"10": {
+			"label": "Temperature Differential Report",
+			"description": "Differential between current & previous report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "Temperature Differential Report Threshold",
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 16]
```diff
@@ -0,0 +1,158 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 5
+		},
+		"11": {
+			"label": "LED Blink Enable",
+			"description": "Defines whether the LED blinking is on / off",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "LED: Blink on Motion",
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable LED blinking",
+					"value": 0
+				},
+				{
+					"label": "Enable LED blinking",
+					"value": 1
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
```

File: packages/config/config/devices/0x0258/motion_sensor_2.json [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+			]
+		},
+		"99": {
+			"label": "Ambient light intensity calibration",
+			"description": "Define the calibrated scale for ambient light intensity",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/motion_sensor_2.json]:
```suggestion
			"label": "Ambient Light Intensity Calibration",
```
I don't understand the manual here 🤷🏻‍♂️

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by bwosborne2** [comment]
Sorry, I based the new file off the old one, assuming it had passed your checks.

Also, I could not find out how to fix the semantic title / commit message check. Where is that documented?

**Comment by AlCalzone** [comment]
> assuming it had passed your checks

Many of our files were automatically imported and functionally validated quite some time ago. The style guide is pretty new and requires manual intervention. We apply it as we go, so many files aren't up to date yet.

> Also, I could not find out how to fix the semantic title / commit message check

I did that for you. The message must be in the format
```
changetype(config): lowercase Rest of the SENTENCE
                    ^^^^^^^^^
```

**Comment by AlCalzone** [comment]
I think you overlooked those - did it for you:
grafik

**Review by AlCalzone — APPROVED** [review]

**Comment by bwosborne2** [comment]
Thanks. I thought I got them all...

**Pull request merged**